model/messages/collection.go

```diff
@@ -41,8 +41,9 @@ type ClusterBlockVote struct {
 }
 
 // ClusterTimeoutObject is part of the collection cluster protocol and represents a collection node
-// timing out in given round.
+// timing out in given round. Contains a sequential number for deduplication purposes.
 type ClusterTimeoutObject struct {
```

Comment:
This and `ClusterBlockVote` are identical to their consensus equivalents. I'm not sure there's a benefit to defining them separately (or even having different types, really). 

We could alias them to avoid duplicate declarations, or just use the consensus message types in cluster consensus as well. 
```suggestion
type ClusterTimeoutObject TimeoutObject
```

Reply:
I was always under the impression that we actually want to have different types to be able to distinguish between cluster and main consensus messages. I believe we have talked about this once and decided to have different messages, haven't we?